src/params/geo-point.js

```diff
@@ -20,6 +24,9 @@ class GeoPoint extends NumberList {
       ) {
         valid = false;
       }
+      if (this.relaxed === false && (valueParsed[0] === 0 || valueParsed[1] === 0)) {
```

Comment:
fix, and explain to me why it needs fixing :D

Reply:
if valid was false from previous check it is redundant to do this check.

Quick thought. Should we refactor the validate in "geo-point.js" and "geo-rect.js" to be more like "str-list.js"?

Reply:
It's actually much worse than just inefficient. Know why?

Yeah, let's do that